pages/interview-process.md

```diff
@@ -3,62 +3,43 @@ permalink: /interview-process/
 title: Interview process
 ---
 
-### Phone screen
+### Application review
 
-The first step in the interview process is the phone screen, a 30-minute phone call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here.
+The first step of the process will be a review of the information that you have submitted as part of your application. Your application will be evaluated against the Qualification Competencies detailed below.
 
-### In-person (or videochat) interviews
-
-After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews. Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday. 
-
-### Basic rubric and criteria
```

Comment:
@rooneywp Has this been supplanted by the "Application qualification competencies" section? If so, let me know and I (or someone else) can add it to the engineering interviews page per #105.